### PR TITLE
Add vsdbg for .net core

### PR DESCRIPTION
**Pull request by quoctruong**
PTAL @briandealwis

### CODE REVIEW

File: skaffold-duct-tape/duct-tape/Dockerfile [hunk 1]
```diff
@@ -19,3 +24,4 @@ WORKDIR /duct-tape
 COPY --from=delve /go/bin/dlv go/bin/
 COPY --from=python27 /ptvsd/ python/
 COPY --from=python37 /ptvsd/ python/
+COPY --from=netcore /vsdbg/ netcore/
```
Comment (briandealwis) [inline skaffold-duct-tape/duct-tape/Dockerfile:27]: Will vsdbg support future versions like .NET 5?  Should we version this directory?

Comment (quoctruong) [reply skaffold-duct-tape/duct-tape/Dockerfile:27]: `vsdbg` seems to work with all versions of .NET Core so far (1.0, 2.0 and soon to be released 3.0) so I don't think that will be a problem.

Comment (quoctruong) [reply skaffold-duct-tape/duct-tape/Dockerfile:27]: That being said, I just realized `vsdbg` is runtime specific so this version of `vsdbg` will only work for Linux container.

**Review by briandealwis — APPROVED** [review]
LGTM

**Comment by quoctruong** [comment]
Thanks!

File: skaffold-duct-tape/duct-tape/Dockerfile (hunk 1 above)
Comment (briandealwis) [reply skaffold-duct-tape/duct-tape/Dockerfile:27]: That's true of the python support files too.  We'll have to publish separate images when/if we support alternative os/arch combinations.

**Pull request merged**